src/pages/Tasks/components/TaskModal.tsx

```
@@ -0,0 +1,52 @@
+const TaskModal = ({ data, visible, onClose, onChangeTask }: Props) => {
+  return (
+    <Modal visible={visible} onClose={onClose} width={1000}>
+      <div className="w-full h-full">
+        <div className="w-full p-2 bg-gray-200 mb-4">Header</div>
+        <div className="grid grid-cols-12 h-[calc(90vh-150px)]">
+          <div className="col-span-8 mr-4 overflow-y-auto no-scrollbar">
+            <span className="text-2xl font-semibold">{data.summary}</span>
+            <div className="w-full mt-4 flex flex-col gap-2">
+              <label htmlFor="description">Description</label>
+              {data.description && data.description.length > 0 ? (
+                <div dangerouslySetInnerHTML={{ __html: data.description }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)